src/fides/cli/commands/system_scanner.py

```diff
@@ -0,0 +1,181 @@
+@pass_context
+def system_scanner(ctx: Context) -> None:
+    """
+    Commands to use and manage the System Scanner. Requires fidesctl-plus.
+    """
+
+
+@system_scanner.command(name="deploy")
+@pass_context
+@option(
+    "--annotation",
+    "-a",
+    help=(
+        "Custom annotation to apply to the deployed resources, in key=value format. "
+        + "This option may be included multiple times; once for each desired annotation."
+    ),
+    multiple=True,
+)
+@option(
+    "--cluster-name",
+    "-c",
+    "cluster_name",
+    help="The name of your cluster. Otherwise, the name will be taken from the current kubeconfig.",
+    type=str,
+)
+@option(
+    "--deploy-olm",
+    "-o",
+    is_flag=True,
+    help=(
+        "Whether to deploy Operator Lifecycle Management (OLM). OLM is required. "
+        + "Omit this argument only if OLM has already been deployed on the cluster manually, or through another application. "
+        + "OLM is deployed by default on Openshift clusters."
+    ),
+)
+@dry_flag
+@option(
+    "--kubeconfig",
+    "-k",
+    "kubeconfig_path",
+    help=(
+        "Absolute path to the kubeconfig file. "
+        # Intentionally not a `click` default value, to prevent populating it when left unset by the user.
+        + f"[default: {path.expanduser('~')}/.kube/config]"
+    ),
+    show_default=True,
+    type=str,
+)
+@option(
+    "--label",
+    "-l",
+    help=(
+        "Custom label to apply to the deployed resources, in key=value format. "
+        + "This option may be included multiple times; once for each desired label. "
+        + f"The following labels are reserved for internal use, and may not be used: {', '.join(RESERVED_LABELS)}."
+    ),
+    multiple=True,
+)
+@option(
+    "--use-etcd-operator",
+    "-e",
+    is_flag=True,
+    help=(
+        "Whether to use the operator for etcd instead of the statefulset. "
+        + "Required for clusters that do not support persistent volumes."
+    ),
+)
+@verbose_flag
+@yes_flag
+@with_px_auth
+@with_analytics
+def system_scanner_deploy(
+    ctx: Context,
+    annotation: Tuple,
+    label: Tuple,
+    cluster_name: str = "",
+    deploy_olm: bool = False,
+    dry: bool = False,
+    kubeconfig_path: str = "",
+    use_etcd_operator: bool = False,
+    verbose: bool = False,
+    yes: bool = False,
+) -> None:
+    """
+    Deploy the required pods to a Kubernetes cluster.
+    """
+
+    try:
+        annotations = ""
+        if len(annotation):
+            annotations = "--annotations=" + ",".join(annotation)
+
+        labels = ""
+        if len(label):
+            validate_labels(label)
+            labels = "--labels=" + ",".join(label)
+
+        args = [
+            annotations,
+            "--check_only" if dry else "",
+            f'--cluster_name="{cluster_name}"' if cluster_name else "",
+            "--deploy_olm" if deploy_olm else "",
+            f'--kubeconfig="{kubeconfig_path}"' if kubeconfig_path else "",
```

Comment:
I'm currently hitting a wall here. Obviously the user's `~/.kube/config` file is unavailable in the container, and we can't programmatically copy it in. Even if we could, it would be somewhat dangerous to do that.

When I manually add my personal config file to the container using `docker cp`, this command still throws an error:
```
Check pre-check has failed. To bypass pass in --check=false. error=Get "{{ REDACTED }}.eks.amazonaws.com/api/v1/nodes": getting credentials: exec: executable aws not found

It looks like you are trying to use a client-go credential plugin that is not installed.

To learn more about this feature, consult the documentation available at:
      https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins
Pod deployment failed
```

I don't know how we get around this. The command doesn't fail locally because [I assume] I also have the AWS CLI installed locally. We could install the AWS CLI in the container, but that would be a lot of bloat for non-plus users. Also, it would only cover users attempting to use the system scanner with clusters running in AWS.

Reply:
I'm not sure if it would solve the issue, and I'm not sure how much lighter it would be, but maybe it would work to just use the [aws-iam-authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html) instead of the full AWS CLI.

This would still need the config file to get the login info though so if it solves anything at all it would only be part of it.

Reply:
if we can restrict this dependency to plus, that seems reasonable? Having those CLIs integrated is the cost of doing business I think at this point

As for the kubecontext, can we somehow prompt the user to either copy it in, set it as an env var, or something of that nature?